settings.gradle.kts

```diff
@@ -1,12 +1,26 @@
 rootProject.name = "gifdecoder"
 
-dependencyResolutionManagement {
```

Comment:
Hum, how could that be deprecated, as it got added in gradle 6.8? https://docs.gradle.org/6.8/release-notes.html

Reply:
See Central declaration of repositories